packages/flutter/test/material/dropdown_form_field_test.dart

```diff
@@ -715,4 +715,52 @@ void main() {
     expect(value, equals('two'));
     expect(dropdownButtonTapCounter, 2); // Should not change.
   });
+
+  testWidgets('DropdownButtonFormField should re-render if value param changes', (WidgetTester tester) async {
+    String currentValue = 'two';
+    final GlobalKey<FormFieldState<String>> key = GlobalKey<FormFieldState<String>>();
+
+    await tester.pumpWidget(
+      StatefulBuilder(
+        builder: (BuildContext context, StateSetter setState) {
+          return MaterialApp(
+            home: Material(
+              child: DropdownButtonFormField<String>(
+                key: key,
+                value: currentValue,
+                hint: const Text('Select Value'),
+                decoration: const InputDecoration(
+                    prefixIcon: Icon(Icons.fastfood)
+                ),
+                items: menuItems.map((String value) {
+                  return DropdownMenuItem<String>(
+                    value: value,
+                    child: Text(value),
+                    onTap: () {
+                      setState(() {
+                        currentValue = value;
+                      });
+                    },
+                  );
+                }).toList(),
+                onChanged: (String _) {
+                  // Ignored
+                },
```

Comment:
I think `onChanged` does not need to be defined for the `DropdownButtonFormField` to be enabled. If that's the case, we can remove this parameter as well.

Reply:
Sorry I didn't get this

Reply:
Ahhh. Sorry I was confused. I get your point. The problem is that this param is required that is why I didn't removed it.

Do you think that I should (// ignore:) the warning?

Reply:
I passed the default value created at the start of the file!

Reply:
That works! I had assumed that `onChanged` did not need to be defined for the button to be enabled, so this was my mistake as well